source/base/image/hdr.cpp

```diff
@@ -132,7 +132,7 @@ void ReadOldLine(unsigned char *scanline, int width, IStream *file)
 
         // NB EOF won't be set at this point even if the last read obtained the
         // final byte in the file (we need to read another byte for that to happen).
-        if(*file == false)
+        if((bool)(*file) == false)
```

Comment:
This commit definitely has issues.  My apologies.

The only thing left that appears to be any issue is this specific line, and not because of C++11.  (I'm really not sure what I was thinking...)

Anyways, I am going to replace it with a commit to handle just that one issue, since the rest doesn't make very much sense.